src/PrestaShopBundle/Resources/config/routing/admin/improve/design/positions.yml

```diff
@@ -14,7 +14,7 @@ admin_modules_positions_edit:
 
 admin_modules_positions_unhook:
     path: /unhook
-    methods: [POST]
+    methods: [POST,GET]
```

Comment:
I'm not really fan of using GET and POST methods on the same uris, this will make the authorization system harder to setup, don't you think?